modules/postgres/postgres_test.go

```diff
@@ -171,6 +209,54 @@ func TestWithConfigFile(t *testing.T) {
 	defer db.Close()
 }
 
+func TestWithSSL(t *testing.T) {
+	ctx := context.Background()
+
+	caCert, serverCerts, err := createSSLCerts(t)
+	require.NoError(t, err)
+
+	container, err := postgres.RunContainer(ctx,
```

Comment:
suggestion: let's name the variable `ctr` to avoid potential conflicts with a `container` package (docker, moby exposes it)
```suggestion
	ctr, err := postgres.RunContainer(ctx,
```